src/Angular.js

```diff
@@ -125,6 +127,30 @@ var VALIDITY_STATE_PROPERTY = 'validity';
 
 var hasOwnProperty = Object.prototype.hasOwnProperty;
 
+var OBJECT_MAX_DEPTH_IN_ERROR_MESSAGE = 5;
+
+/**
+ * @ngdoc function
+ * @name angular.configureMinErr
+ * @module ng
+ * @kind function
+ *
+ * @description Use this function to configure minErr
+ *
+ * @param {Object=} config an object for defining configuration for the minErr. The
```

Comment:
`minErr` is private/undocumented, so most users won't know what it is. Change to something more descriptive.